jupyterlab/gst/ui-tests/README.md

```diff
@@ -0,0 +1,117 @@
+# Test
+
+The test will produce a video to help debugging and check what happened.
+
+To execute integration tests, you have two options:
+
+- use docker-compose (cons: needs to know and use docker) - this is a more reliable solution.
+- run tests locally (cons: will interact with your JupyterLab user settings)
+
+## Test on docker
+
+1. Compile the extension:
+
+```
+jlpm install
+jlpm run build:prod
+```
+
+2. Execute the docker stack in the example folder:
+
+```
+docker-compose -f ../end-to-end-tests/docker-compose.yml --env-file ./ui-tests/.env build
```

Comment:
Where is this `end-to-end-tests` folder?

Reply:
This is a part of the cookiecuter project for the extension. No e2e folder in this extension. Do you think I should clean this up now?

Reply:
I'm happy with this now. But could we add a note for that?

Reply:
Cleaned up the docker section from the test docs